rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/ecc/SW360EccService.java

```diff
@@ -0,0 +1,212 @@
+/*
```

Comment:
Change filename to EccService.java and update licenseHeader year in newly created files.